x-pack/plugins/elastic_assistant/server/routes/post_actions_connector_execute.ts

```diff
@@ -255,6 +256,7 @@ export const postActionsConnectorExecuteRoute = (
 
           const elserId = await getElser(request, (await context.core).savedObjects.getClient());
 
+          const llmType = getLlmType(connectorId, connectors);
```

Comment:
`getLlmType` is wrong. It infers `llmType` from provider. Therefore, it will assign Azure OpenAI as Bedrock. 

`getLlmType` should be refactored to take `actionTypeId` as the param. then...

```
const getLlmType = (actionTypeId: string) => llmTypeDictionary[actionTypeId];
```